src/common/include.am

```diff
@@ -54,6 +54,7 @@ COMMONHEADERS = \
   src/common/compat_libevent.h			\
   src/common/handles.h				\
   src/common/procmon.h				\
+  src/common/socks5_status.h                    \
```

Comment:
To respond to the commit message, my guts feeling tells me that we probably want a `socks5` library at some point for which it should be (I think!) only visible to the "tor client" component.

Reply:
Yeah, I agree